language-server/test/dotty/tools/languageserver/CompletionTest.scala

```diff
@@ -106,15 +114,16 @@ class CompletionTest {
   }
 
   @Test def importCompleteIncludesSynthetic: Unit = {
+    val expected = Set(
```

Comment:
Personally I'm not a big fan of putting the expected values before the code snippets. IMHO this makes the tests harder to read (to understand what a test is about I read the snippet first and then Ι go back to the expected results) so I would rather leave expected values inlined at the end. However to make things more readable than currently we could:
* unify formatting (e.g. don't put multiple expected values in the same line)
* get rid of the very repetitive `code"""...""".withSource` pattern (e.g. by adding an implicit conversion)
* overload `completion` to accept varargs so that we get rid of `Set(...)` - also we could make `m1` default as in most cases we need only one code marker
After these changes this test could look something like
```scala
  @Test def importCompleteIncludesSynthetic: Unit = {
    code"""case class MyCaseClass(foobar: Int)
           object O {
             val x = MyCaseClass(0)
             import x.c${m1}
           }"""
      .completion(
        ("copy", Method, "(foobar: Int): MyCaseClass"),
        ("canEqual", Method, "(that: Any): Boolean")
      )
  }
```

Reply:
I totally agree. I made quite a makeover there but all language server tests can now omit `.withSource`. Furthermore completions now take marker `m1` as default one and allows completions to be passed as varargs.